src/MICore/MICore.csproj

```diff
@@ -13,17 +13,21 @@
     <AssemblyName>Microsoft.MICore</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
```

Comment:
If you edit a .resx file after this change, does the designer do the right thing?